packages/sdk-core/tests/converters/stackTraceConverterTests.spec.ts

```diff
@@ -0,0 +1,152 @@
+import { StackTraceConverter } from '../../src/modules/converter/StackTraceConverter';
```

Comment:
You should add tests with anonymous functions.

If we're trying to build a robust solution, I would start with supporting all cases listed here:
https://github.com/stacktracejs/error-stack-parser/blob/master/spec/fixtures/captured-errors.js
(the code is MIT, so I think you could just copy the sample stacks)